samples/identify-graphics/src/main/java/com/esri/arcgismaps/sample/identifygraphics/components/MapViewModel.kt

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.identifygraphics.components
+
+import android.app.Application
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.compose.ui.unit.dp
+import com.arcgismaps.Color
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.Polygon
+import com.arcgismaps.geometry.PolygonBuilder
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.symbology.SimpleFillSymbol
+import com.arcgismaps.mapping.symbology.SimpleFillSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.view.Graphic
+import com.arcgismaps.mapping.view.GraphicsOverlay
+import com.arcgismaps.mapping.view.ScreenCoordinate
+import com.arcgismaps.toolkit.geoviewcompose.MapViewProxy
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.launch
+
+class MapViewModel(app: Application) : AndroidViewModel(app) {
+
+    // Create a polygon from a list of points
+    private val polygon: Polygon = PolygonBuilder(SpatialReference.webMercator()).apply {
+        addPoint(Point(x = -20e5, y = 20e5))
+        addPoint(Point(x = 20e5, y = 20e5))
+        addPoint(Point(x = 20e5, y = -20e5))
+        addPoint(Point(x = -20e5, y = -20e5))
+    }.toGeometry()
+
+    // ArcGISMap displayed by the MapView. Use a topographic basemap.
+    val arcGISMap by mutableStateOf(
+        ArcGISMap(BasemapStyle.ArcGISTopographic).apply {
+            // Set an initial viewpoint with the polygon graphic centered on the screen.
+            initialViewpoint = Viewpoint(
+                center = polygon.extent.center,
+                scale = 1.3e8
+            )
+        }
+    )
```

Comment:
`arcGISMap` doesn't need to use `mutableStateOf` here: 
```suggestion
    val arcGISMap = ArcGISMap(BasemapStyle.ArcGISTopographic).apply {
        // Set an initial viewpoint with the polygon graphic centered on the screen.
        initialViewpoint = Viewpoint(
            center = polygon.extent.center,
            scale = 1.3e8
        )
    }
```